src/auditwheel/repair.py

```diff
@@ -79,16 +84,35 @@ def repair_wheel(
                         % soname
                     )
 
+                if not copy_site_libs and "site-packages" in str(src_path).split(
```

Comment:
There's no guarantee for the `site-packages` folder to have that name (in fact, it doesn't on debian python, it's name is `dist-packages`).